### ISSUE TITLE
Chicken Shedder 1.2.3 fails to load in older versions of Minecraft

### ISSUE THREAD

**Issue by kccricket** [Vaatik/chicken-shedder#1]
Chicken Shedder version 1.2.3 fails to load in Minecraft 1.19.4, Paper build 550.

``` plaintext
[15:17:35 ERROR]: [ModernPluginLoadingStrategy] Could not load plugin 'ChickenShedder-[1.2.3].jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: Unsupported API version 1.20
```

I'm no plugin developer, but you've set `api-version: '1.20'` in the plugin.yml file. By setting this, you're saying that the plugin requires API features not available in MC versions before 1.20. Since the Spigot project page states it's been tested with 1.17, I assume it requires at least 1.17. If so, I recommend setting `api-version: '1.17'`. It's not necessary to update `api-version` with each new version of Minecraft.

Love the plugin, BTW. It's always bothered me that you have to kill chickens to get feathers.

**Comment by Vaatik** [comment]
You are absolutely right, thank you for the feedback I will look into it soon :)

**Comment by Vaatik** [comment]
> Chicken Shedder version 1.2.3 fails to load in Minecraft 1.19.4, Paper build 550.
> 
> ```
> [15:17:35 ERROR]: [ModernPluginLoadingStrategy] Could not load plugin 'ChickenShedder-[1.2.3].jar' in folder 'plugins'
> org.bukkit.plugin.InvalidPluginException: Unsupported API version 1.20
> ```
> 
> I'm no plugin developer, but you've set `api-version: '1.20'` in the plugin.yml file. By setting this, you're saying that the plugin requires API features not available in MC versions before 1.20. Since the Spigot project page states it's been tested with 1.17, I assume it requires at least 1.17. If so, I recommend setting `api-version: '1.17'`. It's not necessary to update `api-version` with each new version of Minecraft.
> 
> Love the plugin, BTW. It's always bothered me that you have to kill chickens to get feathers.

Can you tell me if you go to the version history of the plugin and use the version 1.2.2 if it solves your issue. The error you are getting is that the plugin last version (updated for minecraft 1.20) is not compatible with your server's spigot jar 1.19.4. 

Using the version 1.2.2, which supports any spigot jars from minecraft 1.19 should be working. Here's the link to the plugin version history
https://www.spigotmc.org/resources/chicken-shedder.99449/history

Sorry for the confusion I'm still struggling a bit with the naming conventions of spigot plugins ^^

**Comment by Vaatik** [comment]
I will be closing this issue for now. If you ever run into more problems let me know !
